parsl/executors/high_throughput/interchange.py

```diff
@@ -18,7 +18,7 @@
 
 from parsl.app.errors import RemoteExceptionWrapper
 from parsl.monitoring.message_type import MessageType
-
+# from parsl.executors.high_throughput.mac_safe_queue import MacSafeQueue as queue
```

Comment:
get rid of this line so interchange.py isn't touched by this PR at all

Reply:
Done.